doc/releases/changelog-dev.md

```diff
@@ -65,6 +65,10 @@
 
 <h3>Breaking changes 💔</h3>
 
+* `qml.GateFabric`, `qml.AllSinglesDoubles`, and `qml.BasisRotation` now have the `wires` argument as the last positional argument,
+  which is a requirement for PennyLane operations.
```

Comment:
I'm not sure we have a document that fixes this as a hardcore requirement. We could also call it a convention here, not sure.

Reply:
I was also unsure. I replaced `requirement` with `convention`